Add blur param

sgt0 opened the pull request:
Extend the zimg filters to support an optional blur parameter.

I just put a "2" at the end of their names like in resize"2", but can rename them to something like `BlurrableBilinearFilter`, `BilinearBlurFilter`, or whatever else if we want.

Did not touch the Custom filter, dunno if users of that want to be in control of how blur is done or if we should implement it anyways for convenience.

Sample script I used to test:

```python
import vskernels

src = # ...
scale_args = {"width": src.width * 2, "height": src.height * 2}

set_output(core.placebo.Resample(src, filter="bicubic", param1=0, param2=0.5, **scale_args), "placebo")
set_output(vskernels.Catrom.scale(src, **scale_args), "custom")
set_output(core.resize2.Bicubic(src, filter_param_a=0, filter_param_b=0.5, **scale_args), "resize2")

set_output(core.placebo.Resample(src, filter="bicubic", param1=0, param2=0.5, blur=1.15, **scale_args), "placebo blur")
set_output(vskernels.Catrom.scale(src, blur=1.15, **scale_args), "custom blur")
set_output(core.resize2.Bicubic(src, filter_param_a=0, filter_param_b=0.5, blur=1.15, **scale_args), "resize2 blur")

```

Setsugennoao commented:
since the edits are so simple i'd prefer if you directly modified zimg code and diffed it like i did
also i think it'd be fine if you added blur to custom kernel too

sgt0 commented:
IMO patches should be reserved for when something would be infeasible otherwise, but alright I moved it to a patch. Also added support to Custom as well.

sgt0 commented:
Superseded by 8c3a6be.
